### ISSUE TITLE
Make segments accessible

### ISSUE THREAD

**Issue by andris-zalitis** [enfluid/apple-scrub-bar#7]
Either make `UIImageView`s accessible (with traits `UIAccessibilityTraitButton`), or if it doesn't work, create `UIAccessibilityElement` with a corresponding frame for each `UIImageView`.

**Comment by rudolf-adamkovic** [comment]
@andris-zalitis 

> [...] or if it doesn't work [...]

That should totally work.
